tests/test_readme.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018-2020 earthobservations
+# Copyright (c) 2018-2020 Andreas Motl <andreas.motl@panodata.org>
+# Copyright (c) 2018-2020 Benjamin Gutzmann <gutzemann@gmail.com>
+from pathlib import Path
+import doctest
+
+
+def test_readme():
+    readme_file = Path(__name__).parent / "README.rst"
+
+    failures, _ = doctest.testfile(
+        filename=str(readme_file),
+        module_relative=False,
+    )
+
+    assert failures is 0
```

Comment:
```suggestion
    assert failures == 0
```

It's an equality check, not an identity check.

Reply:
Ah, `flake8` apparently already caught this. Nice!
```
tests/test_readme.py:17:12: F632 use ==/!= to compare constant literals (str, bytes, int, float, tuple)
```